package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "nvs",
-	"version": "1.3.0",
+	"version": "1.3.1",
```

Comment:
altho actually, this is semver-minor imo, not patch, and generally version numbers should never be bumped in PRs